packages/client/src/Layouts/Base/navbar/Navbar.tsx

```diff
@@ -1,16 +1,57 @@
-import React, { ReactNode } from 'react'
-
+import React, { useEffect, ReactNode, FC } from 'react'
+import { BsMoonFill, BsSun } from 'react-icons/bs'
 import styles from './Navbar.module.scss'
+import { useAppDispatch, useAppSelector } from '../../../redux/store'
 
+import { setTheme } from '../../../redux/features/auth/authSlice'
 interface NavbarProps {
   children: ReactNode
 }
 
+type Props = {
+  onClick: () => void
+  theme: string
+}
+
+const ThemeSwtcher: FC<Props> = ({ onClick, theme }) => {
+  return (
+    <button className={styles.switchThemeBtn} onClick={onClick}>
+      {theme === 'light' && (
+        <>
+          <BsSun />
+          light theme
+        </>
+      )}
+      {theme === 'dark' && (
+        <>
+          <BsMoonFill />
+          dark theme
+        </>
+      )}
```

Comment:
_Не обязательно исправлять_

В случае, когда темы возможно всего две, можно немного упростить и написать тут через тернарный оператор

```suggestion
      {theme === 'light' ? (
        <>
          <BsSun />
          light theme
        </>
      )
      : (
        <>
          <BsMoonFill />
          dark theme
        </>
      )}
```

Reply:
не очень нравятся тернарки
они менее читаемы в таких структурах, как по мне